docs/source/testing/mocking.md

```diff
@@ -30,15 +30,19 @@ server.listen().then(({ url }) => {
 });
 ```
 
-> Note: If `typeDefs` has custom scalar types, `resolvers` must still contain the `serialize`, `parseValue`, and `parseLiteral` functions
+> Note: If `typeDefs` has any custom scalar types, your `resolvers` must contain the `serialize`, `parseValue`, and `parseLiteral` functions for each of those custom scalars.
 
-Mocking logic simply looks at the type definitions and returns a string where a string is expected, a number for a number, etc. This provides the right shape of result. By default, when using mocks, any existing resolvers are ignored. See the ["Using existing resolvers with mocks"](#using-existing-resolvers-with-mocks) section below for more info on how to change this behavior.
+Mocking logic looks at the types expected by your schema's fields and returns values based on those types. For example, mocking will return a string where a string is expected, a number for a Int, etc. The shape of your mocked results should always be consistent with the shape of a real result from a successful GraphQL operation.
```

Comment:
"types expected by your schema's fields" -> "type returned by each schema field and returns a default value for that type."
"will return" -> "returns"

Reply:
I think it might be worth putting together a table for default GraphQL scalars and their default mocked values. Do you have an Apollo Server instance you could spin up to obtain those values?